Gems/Meshlets/ASV_GPU_and_CPU_Demo/Readme.txt

```diff
@@ -95,19 +95,23 @@ Epic Nanite 2021: https://quip-amazon.com/-/blob/RKb9AAL3zt8/G7A6Cxor0gvn_WPGltB
 Activision - Geometry Rendering Pipeline architecture 2021: https://research.activision.com/publications/2021/09/geometry-rendering-pipeline-architecture
  
 
-Quick Build and Run Direction
-=============================
-1. Go over the files in this directory - you'd needd to folow the instructions below and copy 
+Quick Build and Run Directions
+==============================
+This is a summary of the steps you'll need to follow, see the additional sections below for details.
+
+1. Go over the files in this directory - you'd need to follow the instructions below and copy 
     the files to various locations. Create a Meshlets branch so you'd do it once only.
 2. Add the gem directory to the file engine.json:
     "external_subdirectories": [
         "Gems/Meshlets",
         ...
-3. Run cmake again
-4. Compile meshoptimizer.lib as static library and add it to the Meshlets gem
-5. Build the ASV solution
-6. Run the standalone ASV and choose the Meshlets demo from the RPI demos sub-menu 
-7. When selecting a model, a secondary CPU meshlet model will be generated and 
+3. Compile meshoptimizer.lib as static library and add it to the Meshlets gem
+4. Update several build configuration files to connect the gem to AtomSampleViewer (ASV)
```

Comment:
These changes in comments make it less obvious than what it was before.  I suggest leave this part as it was and point to the section that explain about the build configuration required (or the files that demonstrate it)

Reply:
I'm not sure what you mean, can you be more specific about why this is less obvious? 

I found the original instructions were hard to follow because this first section was high level and lacked detail, and it wasn't clear that this section was more like a summary. I was jumping around a lot trying to piece things together.

So I added a statement above "this is a summary of the steps you'll need to follow, see the additional sections below for details." Then the outline is some high level steps to give the reader an idea of what they'll be doing. Then they need to go through the remaining sections below to follow the details. My statement like "Update several build configuration files" is just meant to give the rough idea, and below there is a section called "Connecting the Gem to the project folder" where there's a bunch of details that cover this point. Perhaps it would help to rename these bullet points to exactly match the wording of the section titles below?